doc/api/next_api_changes/removals/23XXX-GL.rst

```diff
@@ -15,3 +15,8 @@ The **minimum_descent** has been removed from ``TextArea``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The minimum_descent is now effectively always True.
+
+Axes creation no longer accepts positional arguments other than figure and rect
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass in the explicit keyword arguments desired when creating the axes.
```

Comment:
Wording: explicit keyword arguments does not speak to me. Better: "Pass all other parameters as keyword arguments."